docs/source/tools/trigger-service.rst

```diff
@@ -0,0 +1,123 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Trigger Service
+###############
+
+**WARNING:** The triggers-service is an early access feature that is actively being designed and is subject to breaking changes. The documentation here at this time is limited to basic usage. As more features become available the documentation will be updated to include them. We welcome feedback about the trigger-service on our issue tracker, our forum, or on Slack.
```

Comment:
I would suggest that you copy the header from DAML REPL. In particular, it would be nice to have links to issue tracker, forum and slack.

Reply:
Ok!